roles/network_validated_content/files/health_check.yaml

```
@@ -0,0 +1,12 @@
+---
+- name: Include tasks
+  ansible.builtin.include_tasks: ./health_checks/{{ ansible_network_os.split('.')[2] }}.yaml
+
+- name: Set health checks fact
+  ansible.builtin.set_fact:
+     health_checks: "{{ resource_health | health_check_view(item) }}"
```

```suggestion
     health_checks: "{{ resource_health | health_check_view(action) }}"
```
having default `item`  var could leads to warning as we invoke more tasks with nested looping.